### DISCUSSION TITLE
How to use CompiledBinding in the code

### DISCUSSION THREAD

**Discussion by laolarou726** [AvaloniaUI/Avalonia#10878] [Q&A] [answered]
rt

**Comment by hez2010** [comment]
Set `<AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>` in csproj and then you are ready to go.

**Comment by laolarou726** [comment]
> Set `<AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>` in csproj and then you are ready to go.

Sorry, I mean using `control.Bind` extension method with compiled binding in the c# source files. Sorry I didn't make my self clear.

**Comment by timunie** [comment, marked as answer]
See: https://github.com/AvaloniaUI/Avalonia/discussions/10077#discussioncomment-4813935

**Reply by laolarou726** [reply]
Thanks a lot! Are there any plans to simplify this process?
